plugin-flex-ts-template-v2/src/feature-library/contact-history/utils/ContactsUtil.ts

```diff
@@ -0,0 +1,109 @@
+import * as Flex from '@twilio/flex-ui';
+import { Manager } from '@twilio/flex-ui';
+
+import { addContact, setContactList } from '../flex-hooks/state';
+import { Contact } from '../types';
+import { getMaxContacts } from '../config';
+
+const manager = Manager.getInstance();
+const ContactHistoryKey = 'CONTACT_HISTORY';
+
+class ContactsUtil {
+  getRecentContactsList(): Contact[] {
+    const item = localStorage.getItem(ContactHistoryKey);
+    if (item) {
+      return JSON.parse(item);
+    }
+    return [];
+  }
+
+  initContactHistory = () => {
+    const contactList = this.getRecentContactsList();
+    if (contactList && contactList.length > 0) {
+      manager.store.dispatch(setContactList({ contactList }));
+    }
+  };
+
+  setContactList = (contactList: Contact[]) => {
+    localStorage.setItem(ContactHistoryKey, JSON.stringify(contactList));
+  };
+
+  addContact = (task: Flex.ITask) => {
+    const { taskChannelUniqueName: channel, sid: taskSid, queueName, age: duration } = task;
+    const dateTime = task.dateCreated.toLocaleString('en-US');
```

Comment:
This should instead use the current locale Flex is configured for (can get that via `/utils/configuration/index.ts getUserLanguage()`)